esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -149,6 +204,20 @@ def fix_metadata(self, cubes):
         return cubes
 
 
+class O3(Fix):
+    """Fixes for o3."""
+
+    def fix_metadata(self, cubes):
+        """Convert mass mixing ratios to mole fractions."""
+        for cube in cubes:
+            cube.units = "kg kg-1"
```

Comment:
This change of units does not seem to have any effect since different units are assigned on line 217

Reply:
This was more intended as a comment. Fixed in https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/44040c37027c245e4f42f7f451c132439519812f